meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/start-resin-supervisor

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+source /usr/sbin/resin-vars
+
+SUPERVISOR_IMAGE_ID=$(docker inspect $SUPERVISOR_IMAGE | jq '.[0].Id')
```

Comment:
It might be worth using the `--format` option, although I'm not certain what version of docker it was introduced in